### PR TITLE
Ui

### PR DESCRIPTION
**Pull request by ignacio-ramirez-piriz**
New UI for Home Page

### CODE REVIEW

File: density/static/css/style.css [hunk 1]
```diff
@@ -300,6 +450,30 @@ img {
...
+        margin-bottom: 10px;
+    }
+
+    .searchButton {
+        right: 33px;
+    }
+
+    .searchContainer {
+        padding: 0 15px;
+    }
```
Comment (alanhdu) [inline density/static/css/style.css]: This should be merged with the other `.searchContainer`

File: density/templates/capacity.html [hunk 2]
```diff
@@ -1,31 +1,63 @@
 {% extends "layout.html" %}
 {% block body %}
 <div class="listContainer">
-    <a href="https://itunes.apple.com/us/app/density-columbia-study-spaces/id1065376976">
+    <!-- <a href="https://itunes.apple.com/us/app/density-columbia-study-spaces/id1065376976">
         <img class="app_store" src="{{ url_for('static', filename='img/app_store_badge.svg')}}">
-    </a>
+    </a> -->
```
Comment (alanhdu) [inline density/templates/capacity.html]: Let's just delete this instead of commenting out

File: density/templates/predict.html [hunk 3]
```diff
@@ -9,20 +9,98 @@
 
 <div class="container">
   <div class="col-12"  style="text-align:center;">
-    <h4 style="margin-top: 30px;">Below is the predicted capacity of each building for today: </h4>
+    <h4 style="margin-top: 30px;">Below is the predicted capacity of each building for today.<br><br></h4>
```
Comment (alanhdu) [inline density/templates/predict.html]: If we're going to use `<br/>`, then these should be `<br/>` and should probably outside the `<h4>`. 

Theoretically, though, this should be done via CSS styling

File: density/templates/predict.html [hunk 4]
```diff
@@ -9,20 +9,98 @@
...
+              <button class="clear" onclick="clearFilter('location-filter')">(x)</button>
+          </div>
+
+          <div id="openBtnContainer" align="right">
+              <input type="radio" class="filterTab" name="open-filter" id="open" value="True" onclick = "searchLocations()"/> <label for="open"> Open Now </label>
+              <input type="radio" class="filterTab" name="open-filter" id="closed" value="False" onclick = "searchLocations()"/> <label for="closed">Closed </label>
+              <button class="clear" onclick="clearFilter('open-filter')">(x)</button>
+          </div>
+        </div>
+    </div>
```
Comment (alanhdu) [inline density/templates/predict.html]: If this is just a repeat of above, then we should refactor this into a Jinja2 macro.

File: density/templates/predict.html [hunk 5]
```diff
@@ -9,20 +9,98 @@
...
+      <h4>{{location_name}}</h4>
       <div class='graph'>
         {{ d|safe }}
       </div>
     </li>
     {% endfor %}
   </ul>
 </div>
+<script type="text/javascript">
+  function searchLocations() {
```
Comment (alanhdu) [inline density/templates/predict.html]: Let's move this into a separate file that's included (or just put it into `static/base.js`) and avoid having it twice.

File: density/static/css/style.css [hunk 6]
```diff
@@ -385,13 +559,48 @@ img {
...
+
+    #openBtnContainer{
+        display: inline;
+        width: 100%;
+        padding: 0 10px;
+        position: relative;
+        text-align: center;
+    }
+
```
[the remark this answers is not in the document]
Comment (kennyyuan98) [reply density/static/css/style.css]: I took out redundant styles, but these are needed for mobile (they're within a media query).

File: density/static/css/style.css (hunk 1 above)
Comment (kennyyuan98) [reply density/static/css/style.css]: Same as with the btnContainer styles, we want this padding to occur on .searchContainer only on mobile (it's within the media query).

**Pull request closed without merging**